environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/request/EnvironmentEditRequest.java

```diff
@@ -37,6 +38,9 @@
     @ApiModelProperty(EnvironmentModelDescription.SECURITY_ACCESS)
     private @Valid SecurityAccessRequest securityAccess;
 
+    @ApiModelProperty(EnvironmentModelDescription.IDBROKER_MAPPING_SOURCE)
```

Comment:
As mentioned earlier, I do not see it justified to make any such changes to the Environment service. The decision of whether to generate mock IDBroker mappings or not should depend on the `StackV4Request`, and not driven by environment properties. We can discuss this offline. 😃 Until then, I am not reviewing any Environment related change below.

Reply:
This solution was suggested by and agreed with @doktoric